### PR TITLE
XS✔ ◾ Tiago's changes for better consistency and readability

### PR DESCRIPTION
**Pull request by tiagov8**
cc @leahy268 @Tanya-Leahy @sethdaily 

> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

@adamcogan 's **Re: Nbsp ok?**

> 2. What was changed?

Many things :)

> 3. Did you do pair or mob programming (list names)?

no

### CODE REVIEW

File: rules/use-backchannels-effectively/rule.md [hunk 1]
```diff
@@ -21,75 +21,94 @@ created: 2020-08-31T23:13:15.000Z
...
 :::
 
-If the Software Engineers still disagree after the group chat, then they should make the matter official by sending a ['For the record'](http://www.ssw.com.au/for-the-record) email.
+#### Use "For the record" if disagreement persists
 
-*For example:  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;"For the record… I understand you want microservices because of xxx\
- &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;However I think that is wrong because of yyy\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;I have CCed [3months@followupthen.com](mailto:3months@followupthen.com) to see if any of us have changed our mind"*
+If the Software Engineers still disagree after the group chat, then they should make the matter official by sending a ['For the record'](/for-the-record) email.
```
Comment (leahy268) [inline rules/use-backchannels-effectively/rule.md:63]: Does this violate the principle of DRY?  We already have a rule on 'For the record'.  Won't this mean that in the future we need to update both templates?

Comment (tiagov8) [reply rules/use-backchannels-effectively/rule.md:63]: @leahy268 

From my perspective this is **not** a "for the record" template... but an example.

I personally don't mind removing the example and keeping just the link... but I thought the example was a requirement for this rule.

Comment (leahy268) [reply rules/use-backchannels-effectively/rule.md:63]: Yes I think the template needs to be removed

Comment (tiagov8) [reply rules/use-backchannels-effectively/rule.md:63]: Hi @leahy268 

Done - removed email template to follow DRY

1. Please review again ;)

**Review by leahy268 — APPROVED** [review]
Done

**Pull request merged**